MAINTAINERS.yml

```diff
@@ -1995,7 +1995,9 @@ Release Notes:
     - sample.drivers.espi.ps2
```

Comment:
No objections to the change. Could you change the commit subject and replace "myself" by your GH id. This way it is possible to see immediately who is being added.
So
`MAINTAINERS: add yangbolu1991 as a maintainer for PTP clock drivers`

Reply:
Thanks @jukkar 
I updated it :)